src/stories/4-elements/popover/popover.stories.tsx

```diff
@@ -129,6 +129,29 @@ Default.args = {
 	hint: { content: "Hint" },
 };
 
+export const PopoverImage = Template("popover-image").bind({});
+PopoverImage.args = {
+	uiType: "popover",
+	children: "More info",
+	hint: {
+		content: `<img src='https://assets.life.gov.sg/lifesg/logo-lifesg.svg' alt="logo">`,
+	},
+};
+
+export const PopoverComponent = Template("popover-component").bind({});
```

Comment:
move together with TooltipCustomisation story

```suggestion
export const TooltipCustomisationWithSchema = Template("popover-component").bind({});
```